pkg/operator/provisioner/provisioner.go

```diff
@@ -54,14 +60,15 @@ type provisionerConfig struct {
 	clusterName string
 }
 
-func newRookVolumeProvisioner(clusterManager *clusterManager) controller.Provisioner {
-	return &rookVolumeProvisioner{
-		clusterManager: clusterManager,
+// NewRookVolumeProvisioner creates RookVolumeProvisioner
+func NewRookVolumeProvisioner(client kubernetes.Interface) controller.Provisioner {
```

Comment:
Sure. I feel the type is correct. If i changed it to `Provisioner`, it sounds very generic. But im ok either way.